Templates/Per Tenant Extension/.github/workflows/CreateRelease.yaml

```diff
@@ -302,13 +306,17 @@ jobs:
           ref: '${{ needs.createRelease.outputs.commitish }}'
 
       - name: Create Release Branch
+        env:
+          releaseBranchPrefix: ${{ github.event.inputs.releaseBranchPrefix }}
         run: |
           $errorActionPreference = "Stop"; $ProgressPreference = "SilentlyContinue"; Set-StrictMode -Version 2.0
-          git checkout -b ${{ needs.CreateRelease.outputs.releaseBranch }}
+          $releaseBranch = "$($env:releaseBranchPrefix)" + "${{ needs.CreateRelease.outputs.releaseVersion }}"
```

Comment:
It doesn't need to be in " - could just be $env:xxx

Reply:
Without the `+` it was a bit unreadable. And the quotes `"` cover the case when, for whatever reason, `$($env:releaseBranchPrefix)` is a number.

Reply:
Environment variables are not type strong - they are always strings.